test/e2e/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.20
```

Comment:
minor but we may want to use the same golang version used to build official docker images?

Reply:
Since we are using an older version for building the "production" image, I like using the newer for the tests. We could do the other way around: the point is to be as compatible as possible